app/controllers/admin/instances_controller.rb

```diff
@@ -31,7 +31,7 @@ def ordered_instances
     end
 
     def subscribeable_accounts
-      Account.with_followers.remote.where(domain: params[:by_domain])
+      Account.remote.where(protocol: :ostatus).where(domain: params[:by_domain])
```

Comment:
Note: This query has been wrong for a long time. I think it hasn't been updated since ActivityPub was introduced. It would send PuSH subscriptions to ActivityPub accounts. It also would *not* correctly filter accounts that had local followers. So this is a net improvement.